synadm/cli/__init__.py

```diff
@@ -33,6 +33,15 @@
 from synadm import api
 
 
+output_format_help = """The 'human' mode gives a tabular or list view depending
+on the fetched data, but often needs a lot of horizontal space to display
+correctly. 'json' returns formatted json. 'minified' is minified json, suitable
+for scripting purposes. 'pprint' shows a formatted output with the help of
+Python's built-in pprint module.  'yaml' is a compromise between human- and
```

Comment:
There's an extra space

```suggestion
Python's built-in pprint module. 'yaml' is a compromise between human- and
```